pkg/build/graph/analysis/bc.go

```diff
@@ -346,6 +347,8 @@ func validImageStreamImage(imageNode *imagegraph.ImageStreamImageNode, imageStre
 			}
 		}
 		return false, dockerImageReference.ID
+	} else {
+		fmt.Fprintf(os.Stderr, "ERR(%s): %#+v\n", imageNode.Name, err)
```

Comment:
Is this for debugging purposes? If not, maybe something a bit gentler than "ERR" would be better?